libs/SalesforceSDKCore/SalesforceSDKCore.xcodeproj/project.pbxproj

```diff
@@ -3250,6 +3250,7 @@
 				GCC_WARN_UNUSED_VARIABLE = YES;
 				SDKROOT = iphoneos;
 				SUPPORTED_PLATFORMS = "iphonesimulator iphoneos";
+				SWIFT_COMPILATION_MODE = wholemodule;
```

Comment:
What does this do?

Reply:
Good question - that was the result of clicking `Update to Xcode recommended settings` in `Xcode 10.3`.